src/main/java/dev/isxander/controlify/gui/screen/ControllerConfigScreenFactory.java

```diff
@@ -576,6 +580,17 @@ private ConfigCategory createBindsCategory(ControllerEntity controller) {
         optionBinds.add(new OptionBindPair(radialBind, controller.bindings().RADIAL_MENU));
         category.option(editRadialButton);
         category.option(radialBind);
+        category.option(Option.<Integer>createBuilder()
+                        .name(Component.translatable("controlify.gui.radial_menu.btn_focus_timeout"))
+                        .description(OptionDescription.createBuilder()
+                                .text(Component.translatable("controlify.gui.radial_menu.btn_focus_timeout.tooltip"))
+                                .build())
+                        .binding(def.radialButtonFocusTimeoutTicks * 50,
+                                () -> config.radialButtonFocusTimeoutTicks * 50,
+                                v -> config.radialButtonFocusTimeoutTicks = Math.round(v / 50f)) // round input to nearest 50 to get tick count (50ms = 1 tick)
```

Comment:
Actually, come to think of it, can't you leave the binding alone and just make the formatter `*50`? Sorry to keep messing you around.

Reply:
> Actually, come to think of it, can't you leave the binding alone and just make the formatter `*50`? Sorry to keep messing you around.

Yes, should have done that instead of binding calculations in the first place.